tests/utils/managed_deployment.py

```
@@ -802,23 +868,124 @@ async def _get_pod_status_details(self) -> List[PodStatusDetail]:
-    async def _get_pod_events(self) -> List[str]:
-        """Fetch warning events for pods in this deployment's namespace."""
+    async def _get_pod_names(self) -> List[str]:
+        """Return the names of pods owned by this DGD."""
+        try:
+            assert self._core_api is not None, "Kubernetes API not initialized"
+            label = f"nvidia.com/dynamo-graph-deployment-name={self._deployment_name}"
+            pods = await self._core_api.list_namespaced_pod(
+                self.namespace, label_selector=label
+            )
+            return [p.metadata.name for p in pods.items if p.metadata]
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)